Library/Formula/julius.rb

```diff
@@ -0,0 +1,18 @@
+class Julius < Formula
+  homepage "http://julius.sourceforge.jp"
+  url "http://sourceforge.jp/frs/redir.php?m=iij&f=%2Fjulius%2F60273%2Fjulius-4.3.1.tar.gz"
+  sha1 "88f64ae9ed00b6ab5a2d4fe07e3ced141a46c196"
+
+  depends_on "libsndfile"
+
+  def install
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--mandir=#{man}",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+end
```

Comment:
When I add the following test block (tried other examples found in the Formulas) brew test fails. Any clue as how to resolve this?

```ruby
  test do
    result = pipe_output("#{bin}/julius.dSYM --help")
    assert result.include?("Global Options")
  end
```

This is the output it produces:
```bash
martijn@kenpachi-2 /u/local> brew test julius
Testing julius
==> /usr/local/Cellar/julius/4.3.1/bin/julius.dSYM --help
Julius rev.4.3.1 - based on JuliusLib rev.4.3.1 (fast)

Engine specification:
 -  Base setup   : fast
 -  Supported LM : DFA, N-gram, Word
 -  Extension    : LibSndFile
 -  Compiled by  : clang -g -O2

Options:

--- Global Options -----------------------------------------------

** removed most of long help message **

 Additional options for application:
    [--help]	display this help
    [-help]	display this help
    [-outfile]	save result in separate .out file
    [-nolog]	not output any log
    [-logfile arg]	output log to file
    [-separatescore]	output AM and LM scores separately
    [-kanji arg]	convert character set for output
    [-nocharconv]	disable charconv
    [-charconv arg arg]	convert character set for output
    [-outcode arg]	select info to output to the module: WLPSCwlps
    [-module (arg)]	run as a server module
    [-record arg]	record input waveform to file in dir
Error: julius: failed
Failed assertion, no message given.
martijn@kenpachi-2 /u/local> 
```

I tried some other variants to this found in other formula, but even matching the word Julius seems to fail.